gmaildump/gmailhistory.py

```diff
@@ -233,17 +269,20 @@ def store_msgs_in_db(self, msgs_list):
 
     def get_default_ts(self):
         '''
-        Gives next day date from current day
-        :rtype: str(returns tommorrow date)
+        This fun helps to return next day date from today in Y/m/d format
+
+        :rtype: str
+
         '''
-        self.log.debug('fun get ts with extened time')
+        self.log.debug('get_default_ts')
 
         return (datetime.now() + timedelta(days=1)).strftime('%Y/%m/%d')
 
     def get_history(self, before, after=GMAIL_CREATED_TS):
         '''
         Get all the msgs from the user's mailbox with in given dates and store in the db
-        :Note : Gmail api will consider 'before' : excluded date, 'after' : included date
+        Note : Gmail api will consider 'before' : excluded date, 'after' : included date
```

Comment:
Can we support a doctest, verifying 'Note : Gmail api will consider 'before' : excluded date, 'after' : included date'